src/beautifiers/hh_format.coffee

```diff
@@ -0,0 +1,22 @@
+###
+Requires http://hhvm.com/
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class HhFormat extends Beautifier
+  name: "hh_format"
+  link: "http://hhvm.com/"
+
+  options:
+    PHP: true
+
+  beautify: (text, language, options) ->
+    @run("hh_format", [
+      @tempFile("input", text)
+      ]).then((output) =>
+        # hh_format can exit with status 0 and no output for some files which
+        # it doesn't format.  In that case we just return original text.
```

Comment:
If an error occurred then it should be returned in this Promise for Atom-Beautify to display to the user. Atom-Beautify will then simply not change the text from the original. This is important when debugging, as this would currently not show an error and instead would return the same text, which often is surprising.